src/gc/env/gcenv.base.h

```diff
@@ -394,8 +394,22 @@ template<typename T>
 inline
 T VolatileLoad(T const * pt)
 {
+#if defined(_ARM64_) && defined(__clang__)
+    T val;
+    unsigned lockFreeAtomicSizeMask = (1 << 1) | (1 << 2) | (1 << 4) | (1 << 8);
```

Comment:
A micro nit - static const unsigned ... makes the generated code in debug build shorter by one load and one store to a local variable.